portal/src/main/webapp/js/src/patient-view/util/util.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+/**
+ * Get first key found in object. Otherwise return null.
+ * @param {object} object - object with key/value pairs
+ * @param {array} keys - array of keys
+ */
+window.getFirstKeyFound = function(object, keys) {
+    if (!object) {
+      return null;
+    }
+
+    for (var i=0; i< keys.length; i++) {
+        var value = object[keys[i]];
+        if (typeof value !== 'undefined' && value !== null) {
+          return value;
+        }
+    }
+    return null;
+}
+
+/**
+ * Add format method to JS string, similar to printf. Only added if it isn't
+ * implemented yet.
+ * http://stackoverflow.com/questions/610406/
+ */ 
+if (!String.prototype.format) {
```

Comment:
Really not recommend to attach it into String since you only have one place to use. Suggest to move to patientViewUtil